google-cloud-spanner/test/google/cloud/spanner/client/read_resume_buffer_bound_test.rb

```diff
@@ -184,7 +184,7 @@
       Google::Cloud::Spanner::V1::PartialResultSet.new(results_hash1),
       Google::Cloud::Spanner::V1::PartialResultSet.new(results_hash2),
       Google::Cloud::Spanner::V1::PartialResultSet.new(results_hash3),
-      GRPC::Unavailable,
+      GRPC::Aborted,
```

Comment:
Why do we delete the `GRPC:: Unavailable`? `Aborted` is newly supported. We probably should keep both `Unavailable` and `Aborted`.

Reply:
This test was failing, because we are now retrying `Unavailable` errors, such as in the Java / Python clients. Since this test was not specific to handling `Unavailable` errors, but re-throwing errors in the case of a failure, we changed the error to `Aborted`. Let me know if this makes sense.